aomusicplayer.cpp

```diff
@@ -4,32 +4,23 @@ AOMusicPlayer::AOMusicPlayer(QWidget *parent, AOApplication *p_ao_app)
 {
   m_parent = parent;
   ao_app = p_ao_app;
+  m_player = new QMediaPlayer;
 }
 
 AOMusicPlayer::~AOMusicPlayer()
 {
-  BASS_ChannelStop(m_stream);
+  m_player->stop();
```

Comment:
This is not a QObject, so I believe you need to call `m_player->deleteLater()` yourself on the destructor.